src/Uno.UI/Microsoft/UI/Xaml/Controls/AppBar/AppBarButtonTemplateSettings.cs

```diff
@@ -0,0 +1,23 @@
+
+using Uno;
+using Windows.UI.Xaml;
+
+namespace Microsoft.UI.Xaml.Controls.Primitives
+{
+	public partial class AppBarButtonTemplateSettings : DependencyObject
+	{
+		internal AppBarButtonTemplateSettings()
+		{
+
+		}
+
+		public double KeyboardAcceleratorTextMinWidth
+		{
+			get => (double)GetValue(KeyboardAcceleratorTextMinWidthProperty);
+			internal set => SetValue(KeyboardAcceleratorTextMinWidthProperty, value);
+		}
+
+		internal static DependencyProperty KeyboardAcceleratorTextMinWidthProperty { get; } =
+			DependencyProperty.Register("KeyboardAcceleratorTextMinWidth", typeof(double), typeof(AppBarButtonTemplateSettings), new FrameworkPropertyMetadata(0.0));
```

Comment:
You can use `nameof(KeyboardAcceleratorTextMinWidth)` here for the first parameter